osu.Game/Input/Bindings/GlobalActionContainer.cs

```diff
@@ -111,6 +111,8 @@ public static IEnumerable<GlobalAction> GetGlobalActionsFor(GlobalActionCategory
 
             new KeyBinding(InputKey.F10, GlobalAction.ToggleGameplayMouseButtons),
             new KeyBinding(InputKey.F12, GlobalAction.TakeScreenshot),
+
+            new KeyBinding(InputKey.Insert, GlobalAction.BossKey),
```

Comment:
It looks like this key binding will be visible on all platforms despite it only being supported on Windows? It should only be visible on Windows.

Reply:
![image](https://github.com/user-attachments/assets/d3c64e71-6744-4a86-bbdf-dbe163bc5b1b)

it's a static variable, i'm not sure how to do that

Reply:
Make the getter create an array and then append the windows key binding at the end of it?

Reply:
see https://github.com/ppy/osu/blob/5911c42116f2f1526cc982dcef953ac873a7f9c5/osu.Game/Overlays/Chat/DrawableChatUsername.cs#L157-L185
This should work. and the return type is IEnumerable so there is no need to convert it at the end